R/splitByRanks.R

```diff
@@ -266,12 +266,12 @@ setMethod("unsplitByRanks", signature = c(x = "TreeSummarizedExperiment"),
 }
 
 #' @importFrom SummarizedExperiment assayNames assay
-.unsplit_assays <- function(ses, MARGIN = 1L) {
+.unsplit_assays <- function(ses, by = MARGIN, MARGIN = 1L) {
```

Comment:
Here the name is already handled in a function that calls this function --> no need to modify this function.

We must do as little changes as possible